src/pages/spawn/spawnHost/SpawnHostButton.tsx

```diff
@@ -8,23 +8,24 @@ import { PlusButton } from "components/Buttons";
 import { size } from "constants/tokens";
 import { MyHostsQuery, MyHostsQueryVariables } from "gql/generated/types";
 import { GET_MY_HOSTS } from "gql/queries";
-import { useSpruceConfig } from "hooks";
-import { SpawnHostModal } from "pages/spawn/spawnHost/index";
+import { useSpruceConfig, useUpdateURLQueryParams } from "hooks";
+import { SpawnHostModal } from "pages/spawn/spawnHost/SpawnHostModal";
 import { HostStatus } from "types/host";
 import { queryString } from "utils";
 import { QueryParams } from "../types";
 
 const { parseQueryString } = queryString;
```

Comment:
It was merely to add a space between this declaration and the component declaration below.
```suggestion
const { parseQueryString } = queryString;

```